include/fmt/chrono.h

```diff
@@ -966,13 +966,33 @@ inline void tzset_once() {
 }
 #endif
 
-template <typename OutputIt, typename Char> class tm_writer {
+struct tm_ss {
+  std::tm tm;
+  int subsec_wdt;
+  uint64_t subsec;
+};
+
+struct empty_tm_ss {
+ private:
+  tm_ss tm_ss_;
+  empty_tm_ss() { tm_ss_.subsec_wdt = 0; }
+
+ public:
+  static empty_tm_ss& instance() {
+    static empty_tm_ss inst;
+    return inst;
+  }
+  const tm_ss& get_tm_ss() { return tm_ss_; }
+};
+
+template <typename OutputIt, typename Char> class tm_ss_writer {
```

Comment:
Let's keep the old name since formatting of subseconds is just a minor implementation detail.

Reply:
Kept old name.